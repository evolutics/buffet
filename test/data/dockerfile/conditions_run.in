FROM alpine
ARG example
RUN ls
