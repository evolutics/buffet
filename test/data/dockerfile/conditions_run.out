ARG example=''

ARG _alpine_version='3.9.4'

FROM alpine:"${_alpine_version}"

ARG example
RUN if [[ -n "${example}" ]]; then \
    ls \
  ; fi

WORKDIR /workdir
